lib/form/paragraph3.flow

```diff
@@ -163,6 +182,9 @@ renderParagraph(p : [ParagraphElement], s : [ParagraphStyle], rtl : bool) -> For
 				// It's the safest way to simmulate single line, because reflowParaWords2 does a lot of useful staff.
 				w = if (isSingleLine) 100000.0 else max(0.0, wi);
 				newLines = reflowParaWords(words, w, indent);
+
+				if (hasWidthManager) nextDistinct(widthManager.width, fold(map(newLines, \l -> fold(l.words, 0.0, \acc, w1 -> acc + geParaWordWidth(w1))), 0.0, \acc, width -> if (width > acc) width else acc));
```

Comment:
too long line

Reply:
addressed, reworked to use already calculated width